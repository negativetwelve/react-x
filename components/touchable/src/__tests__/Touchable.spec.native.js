// Modules
import React from 'react';
import Touchable from '../Touchable';

/* eslint-disable no-undef */
describe('Touchable', () => {
  context('with no props', () => {
    itRenders(() => <Touchable children={null} />);
  });

  context('with style prop', () => {
    itRenders(() => <Touchable style={{padding: 12}} children={null} />);
  });
});
