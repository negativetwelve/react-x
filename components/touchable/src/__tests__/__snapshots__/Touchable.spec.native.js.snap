// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Touchable with no props renders snapshot 1`] = `null`;

exports[`Touchable with style prop renders snapshot 1`] = `null`;
